pkg/networks/usernet/client.go

```diff
@@ -74,7 +74,7 @@ func (c *Client) ResolveAndForwardSSH(ipAddr string, sshPort int) error {
 }
 
 func (c *Client) ResolveIPAddress(ctx context.Context, vmMacAddr string) (string, error) {
-	timeout := time.After(2 * time.Minute)
+	timeout := time.After(3 * time.Minute)
```

Comment:
Why does it take more than 2 minutes?

Reply:
During #1625, Why does it take more than 1 minute?

Reply:
Seems rather arbitrary, since it started out as 500ms

Seeing the same type of issues when working with mDNS - sometimes my machine is really slow to respond (like when waking up from idle) - but I also don't want to wait several seconds for when the machine really is gone.

Reply:
I find that the GitHub macOS runners are painfully slow, so I can see that you may need special accommodations for them. Maybe instead of relaxing timeouts generally, we should have some kind of setting, like `LIMA_SLOW_HOST_FACTOR=2` that would double all the timeouts in Lima to be more forgiving?

Feels quite kludgy, but still better than making the machine slow for the regular use case.

Reply:
@SmartManoj Can you try setting `--cpus=1 --memory=1 ` see this helps ??

We are also using macOS runners. 2mins as timeout is working fine. I feel like even after the above 3mins change you will see some failures like this.

Reply:
How reducing the memory will help?

Reply:
Problem with github macos runners is that if it takes more memory they will terminate the vm process / take more time to start basically slowers all other stuff.

In this memory, cpu setup we saw better success rate with macOS runners

Reply:
Still the same; https://github.com/OpenDevin/OpenDevin/actions/runs/9802679379/job/27067695657